app/connector/webhook/src/main/java/io/syndesis/connector/webhook/WebhookConnectorCustomizer.java

```diff
@@ -84,7 +93,7 @@ public void customize(ComponentProxyComponent component, Map<String, Object> opt
     }
 
     private static void removeOutput(final Exchange exchange) {
-        exchange.getOut().setBody("");
+        exchange.getOut().setBody("hh");
```

Comment:
Fishy

Reply:
haha debug code.. I was so focused on getting getting the design document in the right format..